example_test.go

```diff
@@ -18,15 +19,28 @@ func ExampleParse(myToken string, myLookupKey func(interface{}) (interface{}, er
 	}
 }
 
-func ExampleNew(mySigningKey []byte) (string, error) {
+func ExampleNew() {
```

Comment:
To use the Output: style of testing for examples, the signature had to be this.

```
/var/folders/rz/8pj9lzz12fqfx51ddh4x5k813x2t56/T/go-build777895386/github.com/itsjamie/jwt-go/_test/_testmain.go:60: cannot use jwt_test.ExampleNew (type func([]byte) (string, error)) as type func() in field value
```

Without it, this is the error that popped.